tfjs-backend-webgpu/src/kernels/FusedDepthwiseConv2D.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {backend_util, FusedDepthwiseConv2D, FusedDepthwiseConv2DAttrs, FusedDepthwiseConv2DInputs, KernelConfig, KernelFunc, TensorInfo, util} from '@tensorflow/tfjs-core';
+
+import {WebGPUBackend} from '../backend_webgpu';
+import {DepthwiseConv2DProgram} from './depthwise_conv2d_webgpu';
+
+export function fusedDepthwiseConv2D(args: {
+  inputs: FusedDepthwiseConv2DInputs,
+  attrs: FusedDepthwiseConv2DAttrs,
+  backend: WebGPUBackend
+}) {
+  const {inputs, backend, attrs} = args;
+  const {x, filter, bias, preluActivationWeights} = inputs;
+  const {strides, pad, dilations, dimRoundingMode, activation} = attrs;
+
+  let $dilations = dilations;
+  if ($dilations == null) {
+    $dilations = [1, 1];
+  }
```

Comment:
Could this be simply:
```suggestion
  const {strides, pad, dilations = [1, 1], dimRoundingMode, activation} = attrs;
```
?

Reply:
If dilations = null, the default won't apply.

Reply:
Aha, good point, I missed that.